crates/ironrdp-connector/src/credssp.rs

```diff
@@ -180,31 +182,19 @@ impl CredsspSequence {
         }
     }
 
-    pub fn process(&mut self) -> CredsspProcessGenerator<'_> {
-        let request = self.next_request.take().expect("next request"); // FIXME: error handling
```

Comment:
The extended CredSSP sequence was broken by the refactoring done in this PR: https://github.com/Devolutions/IronRDP/pull/260
And it wasn’t enabled by default anymore since https://github.com/Devolutions/IronRDP/pull/131
For this reason, we failed at catching this bug during the refactoring.
This PR is also fixing the CredSSP sequence because we enable it by default again (mstsc does this as well).